Makefile

```diff
@@ -25,3 +28,5 @@ uninstall:
 		/lib/modules/*/extra/hid-viewsonic.ko
 	udevadm control --reload
 	depmod -a
+dist:
```

Comment:
The change is very nice overall, but the commit subject doesn't match. I think it should say something like `Add "dist" make target`.